### PR TITLE
Split out a class that takes care of running ZooKeeper server

### PR DESCRIPTION
**Pull request by hmusum**
Need this to simplify the way this will be done when we start supporting
dynamic reconfiguration of ZooKeeper cluster

### CODE REVIEW

**Review by mpolden — APPROVED** [review]
👍

File: zookeeper-server/zookeeper-server-3.5.6/src/main/java/com/yahoo/vespa/zookeeper/ZooKeeperRunner.java [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+import static com.yahoo.vespa.defaults.Defaults.getDefaults;
+import static com.yahoo.vespa.zookeeper.Configurator.zookeeperServerHostnames;
+
+/**
+ * Writes zookeeper config and starts zookeeper server.
+ *
+ * @author Harald Musum
+ */
+public class ZooKeeperRunner implements Runnable {
+    private static final java.util.logging.Logger log = java.util.logging.Logger.getLogger(ZooKeeperRunner.class.getName());
```
Comment (mpolden) [inline zookeeper-server/zookeeper-server-3.5.6/src/main/java/com/yahoo/vespa/zookeeper/ZooKeeperRunner.java:18]: Nit: Import the `Logger` class.

Comment (hmusum) [reply zookeeper-server/zookeeper-server-3.5.6/src/main/java/com/yahoo/vespa/zookeeper/ZooKeeperRunner.java:18]: Yeah, thanks, copied as-is from other class, will fix in next PR

**Comment by hmusum** [comment]
PR build has not started yet, I give up waiting, merging.

**Pull request merged**